packages/medusa/src/api/routes/admin/collections/add-products.ts

```diff
@@ -28,18 +29,17 @@ import { validator } from "../../../../utils/validator"
  *  "200":
  *    description: OK
  */
-export default async (req, res) => {
+export default async (req: Request, res) => {
```

Comment:
question: Any particular reason as to why we are only typing the request obj?

Reply:
Yeah i will add the description i just didn’t get the time to do all and thing i will stop there for that one 👍 and for the typings there no particular reason i will update that